include/aoapplication.h

```diff
@@ -45,6 +45,10 @@ class AOApplication : public QApplication
   void send_ms_packet(AOPacket *p_packet);
   void send_server_packet(AOPacket *p_packet, bool encoded = true);
 
+  ///////////////server metadata////////////////
+
+  bool ackMS_enabled = false;
+
```

Comment:
People are expected to be using both the proper client and the corresponding matching version of the server to go along with it. This is not needed.